### PR TITLE
fix issue with obtaining the decoder layer number when converting the T5 model.

### PR DESCRIPTION
**Pull request by LitLeo**
### Description
fix issue with obtaining the decoder layer number when converting the T5 model.

### Motivation and Context
fix issue： https://github.com/microsoft/onnxruntime/issues/17072

Test with [byt5-small](https://huggingface.co/google/byt5-small/tree/main) model, which has 12 encoder layers and 4 decoder layers.
Here is the log.
image

### CODE REVIEW

**Comment by LitLeo** [comment]
> @microsoft-github-policy-service agree

@microsoft-github-policy-service agree

File: onnxruntime/python/tools/transformers/models/t5/t5_decoder.py [hunk 1]
```diff
@@ -407,20 +410,21 @@ def verify_onnx(
...
                 max_diff = numpy.amax(numpy.abs(torch_outputs[1][i].cpu().numpy() - ort_outputs[1 + i]))
                 logger.debug(f"self attention past state {i} max_diff={max_diff}")
                 max_diff_all = max(max_diff_all, max_diff)
 
             if isinstance(model, T5DecoderInit):
-                for i in range(2 * model.config.num_layers):
+                for i in range(2 * num_decoder_layers):
                     max_diff = numpy.amax(
-                        numpy.abs(torch_outputs[2][i].cpu().numpy() - ort_outputs[1 + 2 * model.config.num_layers + i])
+                        numpy.abs(torch_outputs[2][i].cpu().numpy() - ort_outputs[1 + 2 * num_decoder_layers+ i])
```
Comment (tianleiwu) [inline onnxruntime/python/tools/transformers/models/t5/t5_decoder.py:427]: Format this line by adding a white space after `num_decoder_layers`. See the python_format pipeline output.

**Review by tianleiwu — APPROVED** [review]

**Pull request merged**